modules/schematics/src/effect/files/__name@dasherize@if-flat__/__name@dasherize__.effects.ts.template

```diff
@@ -38,13 +38,13 @@ export class <%= classify(name) %>Effects {
   <%= effectStart %>
     ofType(<%= classify(name) %>ActionTypes.Load<%= classify(name) %>s),
     /** An EMPTY observable only emits completion. Replace with your own observable API request */
-    concatMap(() => EMPTY)
+    concatMap(() => EMPTY as Observable<{ type: string }>)
```

Comment:
I didn't use Action here because I didn't want to add a reference to `@ngrx/store`